### PR TITLE
Feature/swipescreens

### PR DESCRIPTION
**Pull request by sanderroosendaal**
I added two intro screens as a proof-of-concept. The user gets some basic info to make them excited to use the app.

After you swipe through the screens, you can click Done. Skip Intro state is set to true and user will not see the intro screens any more.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -18,12 +19,9 @@ import { createStore } from 'redux'
 const Stack = createNativeStackNavigator()
 
 function BeginScreen() {
-  const workouts = useSelector((state) => state.workoutReducer.workouts)
   const user = useSelector((state) => state.userReducer)
-  if (workouts.isLoading) {
-    return <SplashScreen />
-  }
-
+  console.log('Signed In', user.isSignedIn)
```
Comment (j-pape) [inline App.js]: plz remove the console logs

File: actions.js [hunk 2]
```diff
@@ -1,5 +1,13 @@
-export default function userSignIn() {
+export function userSignIn() {
+  console.log('signing in')
```
Comment (j-pape) [inline actions.js]: Remove console log

File: screens/IntroScreen.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        </SafeAreaView>
+      </View>
+    )
+  }
+
+  const _onDone = () => {
+    // User finished the introduction. Show real app through
+    // navigation or simply by controlling state
+    dispatch(myActions.finishIntro())
+    console.log('Intro Done')
```
Comment (j-pape) [inline screens/IntroScreen.js]: remove console logs

**Review by j-pape — APPROVED** [review]

File: actions.js (hunk 2 above)
Comment (sanderroosendaal) [reply actions.js]: resolved

**Pull request merged**